flang/lib/Lower/OpenMP.cpp

```diff
@@ -142,6 +142,1122 @@ static void genNestedEvaluations(Fortran::lower::AbstractConverter &converter,
     converter.genEval(e);
 }
 
+//===----------------------------------------------------------------------===//
+// Clauses
+//===----------------------------------------------------------------------===//
+
+namespace detail {
+template <typename C> //
```

Comment:
I usually run `clang-format` from the root directory of the LLVM project, and it puts the following type declaration on the same line with the template, if it fits.  Apparently the lowering code uses MLIR's clang-format settings (which have `AlwaysBreakTemplateDeclarations: Yes`).  The MLIR's format settings are in the mlir subdirectory and are not applied here by default.  The CI format checker expects the MLIR rules, and if I just clang-format myself, it will fail

I'm not sure what exactly is happening here with the formatting, but I put the trailing comment to keep the template header separate even with my settings.

Reply:
Got rid of the trailing comments.